Sources/GoTrue/Types.swift

```diff
@@ -298,7 +298,6 @@ public struct UserIdentity: Codable, Hashable, Identifiable, Sendable {
 }
 
 public enum Provider: String, Codable, CaseIterable, Sendable {
-  case apple
```

Comment:
Not sure I follow why the removal of `apple` as a provider.

Reply:
@ljubiknenad The way I think about this is that we should provide a way for the user to sign in using the `getOAuthSignInURL` even if it makes more sense to use native sign in.

Also removing the `apple` as a provider will throw an error when decoding a user that was signed in using apple, as apple isn't available on the enum.

Reply:
I think to provide a feature-complete library and just warn the user that he shouldn't use the `getOAuthSignInURL` flow to submit to App Store. It's not on us to decide how the user will implement that, we just provide the features and make recommendations for implementation.